src/database/migrations/1.0.0/1696782681632-SystemServicesNode.ts

```diff
@@ -0,0 +1,68 @@
+import {MigrationInterface, QueryRunner, Table} from "typeorm";
+
+export class SystemServicesNode1696782681632 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            "INSERT INTO `fwc_tree_node_types` (`node_type`, `obj_type`, `name`) VALUES( 'SYS', NULL, 'System')"
+        );
+
+        await queryRunner.query(
+            "INSERT INTO `fwc_tree_node_types` (`node_type`, `obj_type`, `name`) VALUES( 'S01', NULL, 'DHCP')"
+        );
+
+        await queryRunner.query(
+            "INSERT INTO `fwc_tree_node_types` (`node_type`, `obj_type`, `name`) VALUES( 'S02', NULL, 'Keepalived')"
+        );
+
+        await queryRunner.query(
+            "INSERT INTO `fwc_tree_node_types` (`node_type`, `obj_type`, `name`) VALUES( 'S03', NULL, 'HAProxy')"
+        );
+
+        const nodes = await queryRunner.query(
+            "SELECT `id` FROM `fwc_tree` WHERE `node_type` IN ('FW', 'CL')"
+        );
+
+        for (const node of nodes) {
+            await queryRunner.query(
+                "INSERT INTO `fwc_tree` (`id_parent`, `name`, `node_type`,`node_order`) VALUES (?, 'System', 'SYS',1)",
+                [node.id]
```

Comment:
What about the fwcloud field? You have to fill with the FWCloud ID to which this node belongs to.
CLUE: You already have it in the firewall or cluster node.

And the id_obj field? It must contain the ID of the firewall or cluster to which this node belongs to.
CLUE: You can find it in the id_obj field of the firewall or cluster node.